modules/client-configuration/pages/autoinst-cdrom.adoc

```diff
@@ -32,11 +32,17 @@ For information about creating a distribution, see xref:client-configuration:aut
 For information about creating a profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
 
 The Cobbler [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO.
-Specifying the distribution with [option]``--distro`` is mandatory.
-For example:
+Specifying the distribution with [option]``--distro`` is mandatory when running [command]``buildiso`` command.
+
+----
+cobbler buildiso --iso=/path/to/boot.iso --distro=<your-distro-label>
+----
+
+To list the names of distributions and profiles stored by Cobbler, run the commands:
```

Comment:
I would clearly mention that you must use distro and profile labels as listed by Cobbler (not by our UI), as this was confusing for customers.

Maybe something like:

```suggestion

NOTE: Keep in mind that you must use distro and profile labels as listed by Cobbler, and not simply as shown in the UI.

To list the names of distributions and profiles stored by Cobbler, run the commands:
```